### ISSUE TITLE
A11Y: Placeholder in UI5 TextArea 

### ISSUE THREAD

**Issue by JosephineSu** [SAP/ui5-webcomponents#2641]
**Describe the bug**

**To reproduce**
1. Go to instance 
2. click goal card go to goalDetail page
3. Apply text spacing CSS through the "stylus" browser extension
4. click Create Activity button
5. Verify the textArea placeholder for text spacing.

How to use stylus: Apply Text spacing settings through "Stylus" browser extension
Follow the instructions from here - https://wiki.wdf.sap.corp/wiki/display/Acc/ACC-283+Details#ACC283Details-TestMethods

**Isolated example**
 https://qacand.lab-rot.ondemand.com/sf/goals?bplte_company=TALH2TGMM1&_s.crb=twWKZOdgcpej8n8NqZh%252fzjwdeC9DDCPUahgy4Q%252b5%252bkE%253d#/goal-detail?targetUserId=cgrant1&templateType=tgm&templateId=777&id=21169
   Company ID: TALH2TGMM1
   Username: cgrant
   passward: Pwd1

**Expected behavior**
TextArea placeholder should adapt the text styling. 
**Screenshots**

![image](https://user-images.githubusercontent.com/38159212/103978020-7a6fa680-51b5-11eb-98c5-f76373ebf832.png)
**Context**
- UI5 Web Components version
- OS/Platform: {...}
- Browser *(if relevant)*: {...}
- Other information: {...}

**Affected components** *(if known)*

**Log output / Any errors** in the console
```
{...}
```

**Organization:** *(if applicable)*

**Priority:** *(optional)* *(Low, Medium, High or Very High)*

Setting priority would indicate the importance of the issue
and would help the process of dispatching and processing.

To set the appropriate priority, you need to consider the impact
and if the issue:
- Breaks entire application or system - High and Very High
- Is accessibility one (Screan Reader, Keyboard Handling) - Medium and High
- Is functional one and makes a single component unusable - Medium and High
- Is visual, styling one - Low and Medium

Note: the priority would be re-evaluated by the issue processors, according to the project internal criterias

**Business impact:** *(mandatory for "Very High")*

**Comment by ilhan007** [comment]
Hello @JosephineSu 

Basically the tool applies the following CSS  to apply letter spacing right?

```css
* { line-height:1.5 !important; letter-spacing:0.12em !important; word-spacing:0.16em !Important; }
```

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-rl this is for you.

Some global styles for letter spacing are not applied to the TextArea value and placeholder

```css
html { line-height:1.5 !important; letter-spacing:0.12em !important; word-spacing:0.16em !Important; }
```

This is not really a bug, normally customizing with global css is not enabled by default, but we try to make components customizable as much as possible.
So, try to set some initial/default values of the browsers for word-spacing, letter-spacing, line-height on the TextArea host and inherit them in the internal elements, so the apply to the placeholder and value texts.

**Comment by JosephineSu** [comment]
Hi @ilhan007 May I know when will this enhancement can be added in UI5-WC?

**Comment by ilhan007** [comment]
Hi @JosephineSu
the enhancement is not planned yet, so we can't give an estimate. The label "approved" means that we will eventually do it and when we label the issue with "Planned" you can expect a change to the master within 2 weeks period and Release Candidate release after 2 more weeks (if you use RC versions) or immediately if you are wiling to use the "preview" versions (like 0.0.0-hash).

BR,
ilhan

**Comment by JosephineSu** [comment]
Got it, thanks!

**Comment by JosephineSu** [comment]
Hi @ilhan007 ,
The refactor didn't apply in V1.0.0-rc.12, neither in v./0.29.x. Please take a look. Thanks a lot.

![image](https://user-images.githubusercontent.com/38159212/110904982-bb667180-8344-11eb-9072-eb60d5c96172.png)

**Comment by ilhan007** [comment]
Fix is released with 0.29.14
